roles/ood/tasks/main.yaml

```diff
@@ -122,3 +122,20 @@
     owner: root
     group: root
     mode: 755
+
+- name: "RUN THESE COMMAND ON OHPC MASTER TO COMPLETE REGISTRATION"
+  pause:
+    seconds: 1
+
+- name: "sudo wwsh node new ood"
+  pause:
+    seconds: 1
+
+- name: "sudo wwsh node set ood -I {{ ood_ip_addr }} --hwadd {{ hostvars[inventory_hostname].ansible_enp0s8.macaddress }}"
```

Comment:
I get what we're trying to accomplish here after reading the discussion but I did not understand how are we setting the MAC addr, in particular, where the MAC addr is coming from in `--hwadd {{ hostvars[inventory_hostname].ansible_enp0s8.macaddress }}`. I'm missing something.

Reply:
Ansible and other configuration management systems gather information about the managed systems in order to make decisions and take actions.  They are set as variables to use in tasks.  Ansible calls these [facts](https://docs.ansible.com/ansible/2.5/user_guide/playbooks_variables.html#information-discovered-from-systems-facts).